.gitignore

```diff
@@ -1,8 +1,9 @@
 # Please try to keep this file as empty as possible,
 # and document what cannot be avoided.
 
-# Give users an ignored place to build
-/build/*
+# Give users an ignored place to build, either a /build/ directory or one with 
+# config parameters such as /build-arm-Debug/
+/build*/*
```

Comment:
Is there a reason that building to `build/arm-Debug` doesn't work? I'd prefer to keep this narrow. I find that without pressure to keep it absolutely minimal, `.gitignore` quickly becomes a dumping ground and impossible to clean.